vaccine_feed_ingest/runners/nv/immunizenevada_org/normalize.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import pathlib
+import sys
+from typing import List, Optional
+
+from vaccine_feed_ingest.schema import schema
+from vaccine_feed_ingest.utils.normalize import provider_id_from_name
+
+
+def _get_id(site: dict) -> str:
+    return f"immunizenevada_org:{site['id']}"
+
+
+def _get_address(address: str) -> schema.Address:
+    parts = address.split(",")
+    street2 = None
+    if len(parts) == 5:
+        street2 = parts[1]
+
+    result = schema.Address(
+        street1=parts[0],
+        street2=street2,
+        city=parts[-3],
+        state=parts[-2],
+        zip=parts[-1],
+    )
+    return result
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
+    contacts = None
+
+    phone = None
+    if "contact-phone" in site:
+        phone = site["contact-phone"]
+
+    # Filter out sites with a url of "/"
+    website = None
+    if site["url"].startswith("http"):
+        website = site["url"]
+
+    if phone or website:
+        general_contact = schema.Contact(
+            contact_type="general",
+            phone=phone,
+            website=website,
+        )
+        contacts = [general_contact]
+
+    return contacts
+
+
+def _get_links(site: dict) -> Optional[List[schema.Link]]:
+    links = []
+    immunize_nv_link = schema.Link(
+        authority="immunizenevada_org",
+        id=site["title"],
```

Comment:
Two notes:
* if we keep this, let's use `id` here
* it is not necessary to include the `Source` location as an `authority` link. We use `authority` to link across datasets - for example if we know that two sites are both `rite_aid` with a Rite Aid provided ID of `123`, we can link them with a higher confidence.

Reply:
Building on this - it looks like this dataset includes a lot of `walgreens`, `walmart`, etc. Would it be possible to build `authority` links for them by parsing the title?

This is okay to move to a followup PR, if you'd like!

Reply:
@eliblock I'm using `provider_id_from_name()`, so I'm adding authority links for `walgreens`, `cvs`, and `safeway`.

I'm missing Walmart and Sam's Club locations, because the regexes don't match the NV naming conventions:

`Sam's Club #6261` doesn't match:
```
m = re.search(r"SAMS PHARMACY 10-(\d+)", name, re.I)
```

`Walmart #3350` doesn't match:
```
m = re.search(r"WALMART PHARMACY 10-(\d+)", name, re.I)
```

Since a change to https://github.com/CAVaccineInventory/vaccine-feed-ingest/blob/main/vaccine_feed_ingest/utils/normalize.py would have wide impact, I agree that it makes more sense to do in a separate PR.

Reply:
I went ahead and removed `immunizenevada_org` as an authority link.

Reply:
Thank you for the detailed explanation here (I should have read the code more closely the first time).